src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -1817,8 +1847,8 @@ export class ChallengeModalBody extends React.Component<ChallengeModalInput, Cha
 
             const settings = {
                 rank: user.ranking,
-                width: this.state.challenge.game.width,
-                height: this.state.challenge.game.height,
+                width: this.state.challenge.game.width ?? -1,
```

Comment:
This change was necessary due to changing width & height to being nullable. With -1 values, no bot is available (as far as I can tell)